src/platform/windows/window.rs

```diff
@@ -1,19 +1,19 @@
 #![cfg(target_os = "windows")]
 
-use std::cell::{Cell, RefCell};
+use std::cell::Cell;
 use std::ffi::OsStr;
-use std::{io, mem, ptr};
 use std::os::windows::ffi::OsStrExt;
-use std::sync::{Arc, Mutex};
 use std::sync::mpsc::channel;
+use std::sync::{Arc, Mutex};
+use std::{io, mem, ptr};
```

Comment:
I prefer to have the top-level brackety imports sorted above more specific ones... this also applies to the `winapi::um` below.